src/aind_data_transfer_models/core.py

```diff
@@ -128,6 +128,25 @@ def get_compress_source_default(
         else:
             return False
 
+    @model_validator(mode="before")
```

Comment:
This is confusing, after testing, I think the issue here is there is a compute_field decorator on the output_folder_name fx on L96. This should be changed to "after" and the type of data is a ModalityConfigs type, not a dict (on my machine after testing). not sure how to delete an attr of the model or if you should?

Reply:
The service de-serializes json, so this will check the json dict for those computed_fields and handle them appropriately.

Reply:
To clarify, I think the fields should still be part of the serialized json object, but validation errors shouldn't be raised when de-serializing those json objects, which is what this PR is trying to accomplish.